js/components/input.jsx

```diff
@@ -23,23 +23,34 @@ var Input = React.createClass({
   },
 
   getInitialState: function() {
-    return {message: ''};
+    return {
+      message: '',
+      commandCharacter: SettingsStore.commandCharacter()
+    };
+  },
+
+  componentDidMount: function() {
+    SettingsStore.on('change', this.settingsStoreChange);
   },
 
   _onChange: function(event) {
-    this.setState({message: event.target.value});
+    this.setState({message: event.target.value, commandCharacter: this.state.commandCharacter});
```

Comment:
let's generally use line breaks whenever objects have more than 1 key/value pair.. there's a few instances of this around